README.md

```diff
@@ -500,96 +441,110 @@ export default MyComponent = () => {
 };
 ```
 
-### `authState`
+## Migrating between versions
 
-Components get this object as a passed prop using the [withOktaAuth](#withoktaauth) HOC or using the [useOktaAuth](#useoktauth) React Hook.  The `authState` object provides synchronous access to the following properties:
-- `.isPending` 
-    - true in the time after page load (first render) but before the asynchronous methods to see if the tokenManager is aware of a current authentication.  
-- `.isAuthenticated`
-    - true if the user is considered authenticated.  Normally this is true if either an idToken or an accessToken is present in the tokenManager, but this behavior can be overridden if you passed an `isAuthenticated` callback to the Security component (or to the Auth instance you passed to the Security component)
-- `.idToken`
-    - the JWT idToken for the currently authenticated user (if provided by the `scopes`)
-- `.accessToken`
-    - the JWT accessToken for the currently authenticated user (if provided by the `scopes`)
-- `.error` 
-    - contains the error returned if an error occurs in `authService.handleAuthentication()` or `authService.updateAuthState()` (which includes any errors encountered when calling the optional `isAuthRequired()` callback provided to `<Security>`)
+### Migrating from 3.x to 4.x
 
-### `authService`
+#### Updating the Security component
 
-Components can get this object as a passed prop using the [withOktaAuth](#withoktaauth) HOC or using the [useOktaAuth](#useoktaauth) React Hook.  The `authService` object provides methods for managing tokens and auth state. 
+From version 4.0, the Security component starts to explicitly accept [oktaAuth][Okta Auth SDK] instance as prop to replace the internal `authService` instance. You will need to replace the [Okta Auth SDK related configurations](https://github.com/okta/okta-auth-js#configuration-reference) with a pre-initialized [oktaAuth][Okta Auth SDK] instance.
```

Comment:
maybe it's worth mentioning that you must add `@okta/okta-auth-js` as a dependency to your project and install it separately from `okta-react`